Fix handling of Google Drive URLs in packaging tools

kmaehashi opened the pull request:
On RHEL 6, `spectool`, which is a tool to download sources used to build RPM packages, does not handle Google Drive URLs correctly.  More precisely:

* It cannot well-handle URLs like ``http:// ... /#/source-0.1.tar.gz``.  The hash part is completely ignored.
* It expects HTTP server that distributes tarball to support time stamping (to check if local copy of tarball is still up-to-date or not.) It seems that Google Drive HTTP servers does not support time stamping, causing ``HTTP 503: Service Unavailable`` error response.

This PR workarounds the problem by avoid using ``spectool``.

rimms commented:
I tested this patch on CentOS 6 and 7. (excute ``package.sh -cuai``)

I think ``wget`` is necessary in the following line.
https://github.com/jubatus/jubatus/blob/d861e86ab5ba19c229213491bb1849c21657fb17/tools/packaging/rpm/package.sh#L121

👍 LGTM except for the comment above.

kmaehashi commented:
@rimms Thanks! Fixed.

As an offline comment from @rimms `which` and `sudo` command is not installed in official CentOS Docker image by default.  I added `sudo` to the list and fixed to avoid using `which` by replacing it with `type` (which is a bash builtin).

rimms commented:
I tested again. It works correctly.👍
